pkgs/by-name/tr/tracetest/package.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, buildNpmPackage
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, testers
+, tracetest
+}:
+
+let
+  pname = "tracetest";
+  version = "0.15.4";
```

Comment:
Looks like they are already on version 1.3.0.
I guess it's better to update as well.